spec/features/form/accessible_autocomplete_spec.rb

```diff
@@ -39,7 +39,7 @@
   end
 
   it "has the correct option selected if one has been saved" do
-    case_log.update!(property_postcode: nil, la: "Oxford")
+    case_log.update!(postcode_known: "No", la: "Oxford")
```

Comment:
If `postcode_known == "No"` doesn't `la` now get reset? And in that case is this working correctly?

Reply:
The LA is not cleared if it was not inferred. Otherwise, we would lose a previously set value.